clang-tools-extra/clangd/InlayHints.cpp

```diff
@@ -867,13 +867,28 @@ class InlayHintVisitor : public RecursiveASTVisitor<InlayHintVisitor> {
     }
   }
 
+  static bool argumentMatchesParamName(StringRef ArgName, StringRef ParamName) {
+    // Exact match.
+    if (ParamName == ArgName)
+      return true;
+
+    // Parameter name uses "a" prefix (e.g. "aParam").
+    if (ParamName.size() > 1 && ParamName[0] == 'a' &&
```

Comment:
I might be complicating things, though shouldn't this also cover `anApple`

Reply:
In the codebase where I encountered this convention, the `a` prefix did not refer to the _word_ "a" (indefinite article), but rather was an abbreviation for "argument" (used as an imprecise synonym for "parameter"). So, with this convention, the prefix would be `a` even in front of `Apple`.

But I suppose I can imagine a different convention where the prefix is an indefinite article and so you'd get `anApple`.